lib/ExpensiMark.js

```diff
@@ -408,7 +408,14 @@ export default class ExpensiMark {
             {
                 name: 'bold',
                 regex: /<(b|strong)(?:"[^"]*"|'[^']*'|[^'">])*>([\s\S]*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
-                replacement: '*$2*',
+                replacement: (match, tagContent, innerContent) => {
+                    const fontWeightMatch = innerContent.match(/style="(.*?)"/);
+                    const isFontWeightBold = fontWeightMatch
+                        ? fontWeightMatch[1].replaceAll(/\s/g, '').includes('font-weight:bold;') || fontWeightMatch[1].replaceAll(/\s/g, '').includes('font-weight:700;')
+                        : false;
+                    const isBold = fontWeightMatch ? isFontWeightBold : true;
```

Comment:
Can you add comments to explain what's happening it became pretty unclear what's going on.